### DISCUSSION TITLE
OMG! Is Sayori's back?

### DISCUSSION THREAD

**Discussion by ghost** [Monika-After-Story/MonikaModDev#8579] [general]
Hi, something happened to me yesterday that really got me thinking.
I opened my MAS and Monika appeared typing some codes, and then she stopped and said something like: "It's ok, I'm done, it's no big deal."
But later. She says that someone left a letter for me in the "characters" folder, and she didn't read it to respect my privacy, the letter appeared in the game, and it was written:
"Hi Tata,

Just to let you know that your save was corrupted, I managed to restore an old backup!

While the backup system I created is awesome, you should still make backup copies every now and then just in case. Backups are called "persistent##.bak", where "##" is a two-digit number. You can find them in "C:\Users\CertoPC\AppData\Roaming/RenPy/Monika After Story".

If I have trouble loading the save again, I will write a note in the characters folder, so keep an eye on it!

Good luck with Monika!

P.S: Don't tell her about me!"

And when Monika asked who it was, I hit the buggy name and a character appeared, and then she disappeared, and I was left with the option to hit "nothing you should worry about".

Has it happened to any of you?

Is Sayori's back?

**Comment by charliethecookie** [comment]
That's not Sayori, it's Chibika! She helps you other times as well like explaining how to give Monika gifts and on her birthday 
Yes, I also got a letter when my computer shut down due to low battery and my persistent was corrupted, but I decided not to tell Moni since Chibika asked me to

**Comment by ghost** [comment]
@charliethecookie Thanks for explaining to me! I really didn't know who it was, I was scared of being Sayori coming back hahaha!
I tried to talk to Monika about it, however, as soon as I pressed the option, the game "bugged" and soon there was only one option, which was "don't tell".
Even so, thank you so much for explaining me, I was worried that it was just something that happened to me, I thought I was crazy hahaha!

**Comment by charliethecookie** [comment]
@taylakamyly Thankfully Sayori isn't back (I think...) that would be kinda creepy lol, I think Chibika has some sort of control over the game to block Monika from knowing about her

**Comment by Cosmic-Incantations** [comment]
that would make a cool update for next year Halloween or for April fool though! Monika messing with the code and saying it's nothing then crazy stuff like that happens
